src/psyclone/tests/psyir/nodes/acc_directives_test.py

```diff
@@ -302,32 +302,6 @@ def test_acckernelsdirective_gencode(default_present):
         "      END DO\n"
         "      !$acc end kernels\n" in code)
 
-
-# (1/1) Method update
-@pytest.mark.parametrize("default_present", [False, True])
```

Comment:
There is an unused import statement (not the fault of this PR)

Reply:
Fixed.